### ISSUE TITLE
[Comments] 170213

### ISSUE THREAD

**Issue by bit101** [bit101/lab#44]
Comments for https://bit101.github.io/lab/dailies/170213.html

**Comment by rpitman** [comment]
Maybe you already saw this:
https://www.youtube.com/watch?v=QAja2jp1VjE

**Comment by bit101** [comment]
yes! saw that a while back. it was in my mind when i did this one.

**Comment by bit101** [comment]
I need to try out the random dot placement and different shape "dots".

**Comment by bit101** [comment]
this too... https://www.youtube.com/watch?v=jMpZ_vpWyWo

**Comment by rpitman** [comment]
3:00 Donald Trump does cryptography.
